files/angular-base-apps/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-base-apps",
+  "repo": "base-apps/angular-base-apps",
+  "files": {
+    "include": ["dist/css/base-apps.css", "dist/css/base-apps.min.css", "dist/js/base-apps.js", "dist/js/base-apps.min.js", "dist/js/base-apps-templates.js", "dist/js/base-apps-templates.min.js"]
```

Comment:
The end result of this config will be that all files are located in the same directory. Without css/ js/. Is this correct?